changelog/fragments/1669212543-improve-error-for-when-instance-lock-is-already-locked.yaml

```diff
@@ -0,0 +1,31 @@
+# Kind can be one of:
+# - breaking-change: a change to previously-documented behavior
+# - deprecation: functionality that is being removed in a later release
+# - bug-fix: fixes a problem in a previous version
+# - enhancement: extends functionality but does not break or fix existing behavior
+# - feature: new functionality
+# - known-issue: problems that we are aware of in a given version
+# - security: impacts on the security of a product or a user’s deployment.
+# - upgrade: important information for someone upgrading from a prior version
+# - other: does not fit into any of the other categories
+kind: enhancement
+
+# Change summary; a 80ish characters long description of the change.
+summary: Adds the file path of the instance lock on the error when it's is already locked
```

Comment:
Beats doesn't use the changelog tool yet so this won't do anything.

You would need to add an entry in CHANGELOG.next.asciidoc at the root of the repo.

We could also just omit a changelog entry for a change this small.